### DISCUSSION TITLE
Notification mails (to multiple admins) aren't sent when moving a spam submission to the inbox

### DISCUSSION THREAD

**Discussion by FrDH** [solspace/craft-freeform#912] [Questions & Answers]
**Describe the bug or issue you're experiencing**
Is a form has multiple admin notifications, the notification mails aren't sent when moving a spam submission to the inbox. It instead throws an error about not being able to find the template (the file it should extend from).

**Steps to reproduce**

1. Create a notification template that extends from another template, basically as describe in the docs.
2. Create a form, choose the created template and add multiple email addresses for the admin notifications.
3. Put the form on a page, go to the page and fill out the form, all specified email addresses should receive the notification mail as expected.
4. Revisit the page and make sure the submission will be marked as spam (we do this by turning off javascript when opening the page)
5. Go to the CP and find the second sumbission in the spam folder, move it to the inbox.
6. Check the mail for the specified email addresses: the second notification is only sent to the first one, all other email addresses don't have the second notification. Also check the freeform dashboard for errors.

**Expected behavior**
The notification mail should probably be sent to all specified email addresses.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.8.15
- Freeform Version: 3.13.31
- Freeform Edition: Pro
- Fresh Install or Upgrade: not really sure what you mean by this

**Additional context**

**Comment by kjmartens** [comment]
I'm very sorry for the delay @FrDH.

I played with this in my testing and cannot duplicate these results. I tried this on Freeform 3 and Freeform 4 and in both cases, all email addresses in the Admin Notify textarea were notified when I approved/allowed the spam submission.

Can you check / try out a few things? 🙂 

1. Create a new formatting template that is fairly standard (nothing special or specific). Assign that as the Admin Notification template instead. Try and see if that makes any difference when you create a new submission flagged as spam.
2. Try selecting ALL in the Admin Notify email address textarea and delete. Then retype the email addresses in there, making sure there are no stray spaces or extra returns.
3. Is this something that has always been happening, or did it start after a recent upgrade on Freeform?

**Comment by FrDH** [comment]
Thanks for coming back to me.
1: I'll try creating a new and standard template next week.
2: I just had a quick test but it doesn't make any difference. 
3: Don't really know.
